src/components/atoms/Timeline/index.tsx

```diff
@@ -22,10 +24,25 @@ export type Props = {
   range?: { [K in keyof Range]?: Range[K] };
   onClick?: TimeEventHandler;
   onDrag?: TimeEventHandler;
+  onPlay?: TimeEventHandler;
+  onOpen?: () => void;
+  onClose?: () => void;
+  isOpened?: boolean;
+  themeColor?: string;
 };
 
 const Timeline: React.FC<Props> = memo(
-  function TimelinePresenter({ currentTime, range, onClick, onDrag }) {
+  function TimelinePresenter({
+    currentTime,
+    range,
+    onClick,
+    onDrag,
+    onPlay,
+    isOpened,
+    onOpen,
+    onClose,
+    themeColor,
```

Comment:
Why do you need a themeColor passed in here? If this is to support the published theme, please refer to the other builtin widgets (Storytelling or Button) for full implementation, but roughly it uses a helper function usePublishTheme and you need to pass the sceneProperty to the component.
```
const publishedTheme = usePublishTheme(sceneProperty?.theme);
```
If this timeline will become an ACTUAL widget, and will be passed in to the Widget component like Storytelling and Button, please also reference that component to see what props you already get with it!

Reply:
This is fixed by [fix: use publishTheme](https://github.com/reearth/reearth-web/pull/277/commits/ba9810cc9acb32d842c134d6ce2253008c2a1824)